src/Bridge.ts

```diff
@@ -29,13 +29,17 @@ import { createAndReturnUser } from './utils/User'
 export function handleCross(event: CrossEvent): void {
   const transaction = createAndReturnTransaction(event)
 
+  const destinationChain = isETHBridge(event.address.toHex()) ? BridgeChain.ETH : BridgeChain.BSC
+
   const crossTransferEvent: CrossTransferEvent = {
     receiver: event.params._to,
     bridgeAddress: event.address.toHex(),
     originalTokenAddress: event.params._tokenAddress,
     amount: event.params._amount,
     decimals: event.params._decimals,
     granularity: event.params._granularity,
+    sourceChain: BridgeChain.RSK,
```

Comment:
It's not necessary to have sourceChain and destinationChain. One chain will always be RSK, why not just have one property called externalChain?

Reply:
done